### PR TITLE
[jrubyscripting] allow disabling dependency tracking

### PR DESCRIPTION
**Pull request by ccutrer**
in case your helper library doesn't support it. or you just don't like the behavior.

### CODE REVIEW

**Comment by jimtng** [comment]
Don't forget to update README

**Comment by boc-tothefuture** [comment]
LGTM pending the answer to @jimtng's questions.

**Comment by ccutrer** [comment]
Sure. Turning off dependency tracking _only_ turns off dependency tracking. The other changes (to the script file watcher) that were made to accommodate the dependency tracker will still be in place.

**Review by jimtng — APPROVED** [review]
LGTM

**Comment by ccutrer** [comment]
@boc-tothefuture : @jimtng hasn't been added to the GitHub Contributors team, so his approval doesn't count as far as GitHub is concerned. Can you plz approve?

**Review by boc-tothefuture — APPROVED** [review]
lgtm

File: bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml [hunk 1]
```diff
@@ -53,12 +56,27 @@
...
+
+		<parameter name="dependency_tracking" type="boolean" required="true" groupName="environment">
+			<label>Enable Dependency Tracking</label>
+			<description>Dependency tracking allows your scripts to automatically reload when one of its dependencies is updated.
+				You may want to disable dependency tracking if you plan on editing or updating a shared library, but don't want all
+				your scripts to reload until you can test it.</description>
+			<options>
+				<option value="true">Enable Dependency Tracking</option>
+				<option value="false">Disable Dependency Tracking</option>
+			</options>
```
Comment (lolodomo) [inline bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml:77]: This is a boolean parameter. No need to define options.

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Review by lolodomo — APPROVED** [review]
LGTM thank you

**Pull request merged**